astropy/units/cgs.py

```diff
@@ -12,7 +12,7 @@
 
 _ns = globals()
 
-def_unit(["cm", "centimeter"], si.cm, namespace=_ns, prefixes=False)
+cm = si.cm
```

Comment:
> One small side effect of this PR that I’ve noticed is that u.cgs.centimeter is now no longer defined; which it previously was. (Of course, u.centimeter and u.si.centimeter both are defined, as before.)

Agreed that we probably shouldn't do this, but `centimeter` can be supported as 

```suggestion
cm = centimeter = si.cm
```

Reply:
My sense is not to add it, given the precedent for `s` and `g`.